README.md

```diff
@@ -1,82 +1,23 @@
 
-A problem arises here, because any database user with CREATEROLE can DROP or ALTER the `connector` role, making your backend service fail.
-From [role attributes docs](https://www.postgresql.org/docs/current/role-attributes.html):
+</p>
 
-> A role with CREATEROLE privilege can **alter and drop other roles, too, as well as grant or revoke membership in them**.
-> However, to create, alter, drop, or change membership of a superuser role, superuser status is required;
-> CREATEROLE is insufficient for that.
+---
 
-The above problem can be solved by configuring this extension to protect the `connector` role:
+**Documentation**: <a href="https://supabase.github.io/supautils" target="_blank">https://supabase.github.io/supautils</a>
 
-```
-supautils.reserved_roles = "connector"
-```
+**Source Code**: <a href="https://github.com/supabase/supautils" target="_blank">https://github.com/supabase/supautils</a>
 
-This extension also allows restricting memberships grants for a set of roles. Certain default postgres roles are dangerous
-to expose to every database user. From [pg default roles](https://www.postgresql.org/docs/11/default-roles.html):
+---
 
-> The pg_read_server_files, pg_write_server_files and pg_execute_server_program roles are intended to allow administrators to have trusted,
-> but non-superuser, roles which are able to access files and run programs on the database server as the user the database runs as.
-> As these roles are able to access any file on the server file system, they bypass all database-level permission checks when accessing files directly
-> and **they could be used to gain superuser-level access**, therefore great care should be taken when granting these roles to users.
+`supautils` is a PostgreSQL extension providing:
 
-For example, you can restrict doing `GRANT pg_read_server_files TO my_role` with:
-
-```
-supautils.reserved_memberships = "pg_read_server_files"
-```
-
-### Installation
-
-Tested to work on PostgreSQL 12 and 13.
-
-Clone this repo and do:
-
-```bash
-make && make install
-```
-
-### Usage
-
-To enable the extension, add on `postgresql.conf`:
-
-```
-shared_preload_libraries = 'supautils'
-```
-
-Protect a list of roles:
-
-```
-supautils.reserved_roles = "supabase_admin,supabase_auth_admin,supabase_storage_admin"
-```
-
-Protect a list of memberships:
-
-```
-supautils.reserved_memberships = "pg_read_server_files, pg_write_server_files, pg_execute_server_program"
-```
-
-### Development
-
-[Nix](https://nixos.org/download.html) is used to set up the environment.
-
-For testing the module you can do:
-
-```bash
-# might take a while in downloading all the dependencies
-$ nix-shell
-
-# test on pg 12
-$ supautils-with-pg-12 make installcheck
-
-# test on pg 13
-$ supautils-with-pg-13 make installcheck
-
-# you can also test manually with
-$ supautils-with-pg-12 psql -U rolecreator
-```
+- The [supabase](https://supabase.io/) SQL standard library
+- Enhanced security for modifying roles
```

Comment:
Added links to the docs. Also reversing the order of the items for now, in case someone(coming from the blog post) wants to see the security stuff right away.
```suggestion
- Enhanced [security](https://supabase.github.io/supautils/role_security/) for modifying roles.
- The Supabase [standard SQL library](https://supabase.github.io/supautils/standard_library/).
```